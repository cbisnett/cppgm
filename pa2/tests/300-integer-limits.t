2147483647 2147483647 2147483647l 2147483647l 2147483647L 2147483647L 2147483647ll 2147483647ll 2147483647LL 2147483647LL 
2147483647u 2147483647u 2147483647ul 2147483647lu 2147483647uL 2147483647Lu 2147483647ull 2147483647llu 2147483647uLL 2147483647LLu 
2147483647U 2147483647U 2147483647Ul 2147483647lU 2147483647UL 2147483647LU 2147483647Ull 2147483647llU 2147483647ULL 2147483647LLU 
2147483647_ud1 2147483647_ud1 2147483647_ud1l 2147483647l_ud1 2147483647_ud1L 2147483647L_ud1 2147483647_ud1ll 2147483647ll_ud1 2147483647_ud1LL 2147483647LL_ud1 
2147483646 2147483646 2147483646l 2147483646l 2147483646L 2147483646L 2147483646ll 2147483646ll 2147483646LL 2147483646LL 
2147483646u 2147483646u 2147483646ul 2147483646lu 2147483646uL 2147483646Lu 2147483646ull 2147483646llu 2147483646uLL 2147483646LLu 
2147483646U 2147483646U 2147483646Ul 2147483646lU 2147483646UL 2147483646LU 2147483646Ull 2147483646llU 2147483646ULL 2147483646LLU 
2147483646_ud1 2147483646_ud1 2147483646_ud1l 2147483646l_ud1 2147483646_ud1L 2147483646L_ud1 2147483646_ud1ll 2147483646ll_ud1 2147483646_ud1LL 2147483646LL_ud1 
2147483648 2147483648 2147483648l 2147483648l 2147483648L 2147483648L 2147483648ll 2147483648ll 2147483648LL 2147483648LL 
2147483648u 2147483648u 2147483648ul 2147483648lu 2147483648uL 2147483648Lu 2147483648ull 2147483648llu 2147483648uLL 2147483648LLu 
2147483648U 2147483648U 2147483648Ul 2147483648lU 2147483648UL 2147483648LU 2147483648Ull 2147483648llU 2147483648ULL 2147483648LLU 
2147483648_ud1 2147483648_ud1 2147483648_ud1l 2147483648l_ud1 2147483648_ud1L 2147483648L_ud1 2147483648_ud1ll 2147483648ll_ud1 2147483648_ud1LL 2147483648LL_ud1 
4294967295 4294967295 4294967295l 4294967295l 4294967295L 4294967295L 4294967295ll 4294967295ll 4294967295LL 4294967295LL 
4294967295u 4294967295u 4294967295ul 4294967295lu 4294967295uL 4294967295Lu 4294967295ull 4294967295llu 4294967295uLL 4294967295LLu 
4294967295U 4294967295U 4294967295Ul 4294967295lU 4294967295UL 4294967295LU 4294967295Ull 4294967295llU 4294967295ULL 4294967295LLU 
4294967295_ud1 4294967295_ud1 4294967295_ud1l 4294967295l_ud1 4294967295_ud1L 4294967295L_ud1 4294967295_ud1ll 4294967295ll_ud1 4294967295_ud1LL 4294967295LL_ud1 
4294967294 4294967294 4294967294l 4294967294l 4294967294L 4294967294L 4294967294ll 4294967294ll 4294967294LL 4294967294LL 
4294967294u 4294967294u 4294967294ul 4294967294lu 4294967294uL 4294967294Lu 4294967294ull 4294967294llu 4294967294uLL 4294967294LLu 
4294967294U 4294967294U 4294967294Ul 4294967294lU 4294967294UL 4294967294LU 4294967294Ull 4294967294llU 4294967294ULL 4294967294LLU 
4294967294_ud1 4294967294_ud1 4294967294_ud1l 4294967294l_ud1 4294967294_ud1L 4294967294L_ud1 4294967294_ud1ll 4294967294ll_ud1 4294967294_ud1LL 4294967294LL_ud1 
4294967296 4294967296 4294967296l 4294967296l 4294967296L 4294967296L 4294967296ll 4294967296ll 4294967296LL 4294967296LL 
4294967296u 4294967296u 4294967296ul 4294967296lu 4294967296uL 4294967296Lu 4294967296ull 4294967296llu 4294967296uLL 4294967296LLu 
4294967296U 4294967296U 4294967296Ul 4294967296lU 4294967296UL 4294967296LU 4294967296Ull 4294967296llU 4294967296ULL 4294967296LLU 
4294967296_ud1 4294967296_ud1 4294967296_ud1l 4294967296l_ud1 4294967296_ud1L 4294967296L_ud1 4294967296_ud1ll 4294967296ll_ud1 4294967296_ud1LL 4294967296LL_ud1 
9223372036854775807 9223372036854775807 9223372036854775807l 9223372036854775807l 9223372036854775807L 9223372036854775807L 9223372036854775807ll 9223372036854775807ll 9223372036854775807LL 9223372036854775807LL 
9223372036854775807u 9223372036854775807u 9223372036854775807ul 9223372036854775807lu 9223372036854775807uL 9223372036854775807Lu 9223372036854775807ull 9223372036854775807llu 9223372036854775807uLL 9223372036854775807LLu 
9223372036854775807U 9223372036854775807U 9223372036854775807Ul 9223372036854775807lU 9223372036854775807UL 9223372036854775807LU 9223372036854775807Ull 9223372036854775807llU 9223372036854775807ULL 9223372036854775807LLU 
9223372036854775807_ud1 9223372036854775807_ud1 9223372036854775807_ud1l 9223372036854775807l_ud1 9223372036854775807_ud1L 9223372036854775807L_ud1 9223372036854775807_ud1ll 9223372036854775807ll_ud1 9223372036854775807_ud1LL 9223372036854775807LL_ud1 
9223372036854775806 9223372036854775806 9223372036854775806l 9223372036854775806l 9223372036854775806L 9223372036854775806L 9223372036854775806ll 9223372036854775806ll 9223372036854775806LL 9223372036854775806LL 
9223372036854775806u 9223372036854775806u 9223372036854775806ul 9223372036854775806lu 9223372036854775806uL 9223372036854775806Lu 9223372036854775806ull 9223372036854775806llu 9223372036854775806uLL 9223372036854775806LLu 
9223372036854775806U 9223372036854775806U 9223372036854775806Ul 9223372036854775806lU 9223372036854775806UL 9223372036854775806LU 9223372036854775806Ull 9223372036854775806llU 9223372036854775806ULL 9223372036854775806LLU 
9223372036854775806_ud1 9223372036854775806_ud1 9223372036854775806_ud1l 9223372036854775806l_ud1 9223372036854775806_ud1L 9223372036854775806L_ud1 9223372036854775806_ud1ll 9223372036854775806ll_ud1 9223372036854775806_ud1LL 9223372036854775806LL_ud1 
9223372036854775808 9223372036854775808 9223372036854775808l 9223372036854775808l 9223372036854775808L 9223372036854775808L 9223372036854775808ll 9223372036854775808ll 9223372036854775808LL 9223372036854775808LL 
9223372036854775808u 9223372036854775808u 9223372036854775808ul 9223372036854775808lu 9223372036854775808uL 9223372036854775808Lu 9223372036854775808ull 9223372036854775808llu 9223372036854775808uLL 9223372036854775808LLu 
9223372036854775808U 9223372036854775808U 9223372036854775808Ul 9223372036854775808lU 9223372036854775808UL 9223372036854775808LU 9223372036854775808Ull 9223372036854775808llU 9223372036854775808ULL 9223372036854775808LLU 
9223372036854775808_ud1 9223372036854775808_ud1 9223372036854775808_ud1l 9223372036854775808l_ud1 9223372036854775808_ud1L 9223372036854775808L_ud1 9223372036854775808_ud1ll 9223372036854775808ll_ud1 9223372036854775808_ud1LL 9223372036854775808LL_ud1 
18446744073709551615 18446744073709551615 18446744073709551615l 18446744073709551615l 18446744073709551615L 18446744073709551615L 18446744073709551615ll 18446744073709551615ll 18446744073709551615LL 18446744073709551615LL 
18446744073709551615u 18446744073709551615u 18446744073709551615ul 18446744073709551615lu 18446744073709551615uL 18446744073709551615Lu 18446744073709551615ull 18446744073709551615llu 18446744073709551615uLL 18446744073709551615LLu 
18446744073709551615U 18446744073709551615U 18446744073709551615Ul 18446744073709551615lU 18446744073709551615UL 18446744073709551615LU 18446744073709551615Ull 18446744073709551615llU 18446744073709551615ULL 18446744073709551615LLU 
18446744073709551615_ud1 18446744073709551615_ud1 18446744073709551615_ud1l 18446744073709551615l_ud1 18446744073709551615_ud1L 18446744073709551615L_ud1 18446744073709551615_ud1ll 18446744073709551615ll_ud1 18446744073709551615_ud1LL 18446744073709551615LL_ud1 
18446744073709551614 18446744073709551614 18446744073709551614l 18446744073709551614l 18446744073709551614L 18446744073709551614L 18446744073709551614ll 18446744073709551614ll 18446744073709551614LL 18446744073709551614LL 
18446744073709551614u 18446744073709551614u 18446744073709551614ul 18446744073709551614lu 18446744073709551614uL 18446744073709551614Lu 18446744073709551614ull 18446744073709551614llu 18446744073709551614uLL 18446744073709551614LLu 
18446744073709551614U 18446744073709551614U 18446744073709551614Ul 18446744073709551614lU 18446744073709551614UL 18446744073709551614LU 18446744073709551614Ull 18446744073709551614llU 18446744073709551614ULL 18446744073709551614LLU 
18446744073709551614_ud1 18446744073709551614_ud1 18446744073709551614_ud1l 18446744073709551614l_ud1 18446744073709551614_ud1L 18446744073709551614L_ud1 18446744073709551614_ud1ll 18446744073709551614ll_ud1 18446744073709551614_ud1LL 18446744073709551614LL_ud1 
18446744073709551616 18446744073709551616 18446744073709551616l 18446744073709551616l 18446744073709551616L 18446744073709551616L 18446744073709551616ll 18446744073709551616ll 18446744073709551616LL 18446744073709551616LL 
18446744073709551616u 18446744073709551616u 18446744073709551616ul 18446744073709551616lu 18446744073709551616uL 18446744073709551616Lu 18446744073709551616ull 18446744073709551616llu 18446744073709551616uLL 18446744073709551616LLu 
18446744073709551616U 18446744073709551616U 18446744073709551616Ul 18446744073709551616lU 18446744073709551616UL 18446744073709551616LU 18446744073709551616Ull 18446744073709551616llU 18446744073709551616ULL 18446744073709551616LLU 
18446744073709551616_ud1 18446744073709551616_ud1 18446744073709551616_ud1l 18446744073709551616l_ud1 18446744073709551616_ud1L 18446744073709551616L_ud1 18446744073709551616_ud1ll 18446744073709551616ll_ud1 18446744073709551616_ud1LL 18446744073709551616LL_ud1 
