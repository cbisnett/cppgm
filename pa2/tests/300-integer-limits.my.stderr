ERROR: Invalid integer literal suffix: 2147483647l_ud1
ERROR: Invalid integer literal suffix: 2147483647L_ud1
ERROR: Invalid integer literal suffix: 2147483647ll_ud1
ERROR: Invalid integer literal suffix: 2147483647LL_ud1
ERROR: Invalid integer literal suffix: 2147483646l_ud1
ERROR: Invalid integer literal suffix: 2147483646L_ud1
ERROR: Invalid integer literal suffix: 2147483646ll_ud1
ERROR: Invalid integer literal suffix: 2147483646LL_ud1
ERROR: Invalid integer literal suffix: 2147483648l_ud1
ERROR: Invalid integer literal suffix: 2147483648L_ud1
ERROR: Invalid integer literal suffix: 2147483648ll_ud1
ERROR: Invalid integer literal suffix: 2147483648LL_ud1
ERROR: Invalid integer literal suffix: 4294967295l_ud1
ERROR: Invalid integer literal suffix: 4294967295L_ud1
ERROR: Invalid integer literal suffix: 4294967295ll_ud1
ERROR: Invalid integer literal suffix: 4294967295LL_ud1
ERROR: Invalid integer literal suffix: 4294967294l_ud1
ERROR: Invalid integer literal suffix: 4294967294L_ud1
ERROR: Invalid integer literal suffix: 4294967294ll_ud1
ERROR: Invalid integer literal suffix: 4294967294LL_ud1
ERROR: Invalid integer literal suffix: 4294967296l_ud1
ERROR: Invalid integer literal suffix: 4294967296L_ud1
ERROR: Invalid integer literal suffix: 4294967296ll_ud1
ERROR: Invalid integer literal suffix: 4294967296LL_ud1
ERROR: Invalid integer literal suffix: 9223372036854775807l_ud1
ERROR: Invalid integer literal suffix: 9223372036854775807L_ud1
ERROR: Invalid integer literal suffix: 9223372036854775807ll_ud1
ERROR: Invalid integer literal suffix: 9223372036854775807LL_ud1
ERROR: Invalid integer literal suffix: 9223372036854775806l_ud1
ERROR: Invalid integer literal suffix: 9223372036854775806L_ud1
ERROR: Invalid integer literal suffix: 9223372036854775806ll_ud1
ERROR: Invalid integer literal suffix: 9223372036854775806LL_ud1
ERROR: decimal integer literal out of range: 9223372036854775808
ERROR: decimal integer literal out of range: 9223372036854775808
ERROR: decimal integer literal out of range: 9223372036854775808l
ERROR: decimal integer literal out of range: 9223372036854775808l
ERROR: decimal integer literal out of range: 9223372036854775808L
ERROR: decimal integer literal out of range: 9223372036854775808L
ERROR: decimal integer literal out of range: 9223372036854775808ll
ERROR: decimal integer literal out of range: 9223372036854775808ll
ERROR: decimal integer literal out of range: 9223372036854775808LL
ERROR: decimal integer literal out of range: 9223372036854775808LL
ERROR: Invalid integer literal suffix: 9223372036854775808l_ud1
ERROR: Invalid integer literal suffix: 9223372036854775808L_ud1
ERROR: Invalid integer literal suffix: 9223372036854775808ll_ud1
ERROR: Invalid integer literal suffix: 9223372036854775808LL_ud1
ERROR: decimal integer literal out of range: 18446744073709551615
ERROR: decimal integer literal out of range: 18446744073709551615
ERROR: decimal integer literal out of range: 18446744073709551615l
ERROR: decimal integer literal out of range: 18446744073709551615l
ERROR: decimal integer literal out of range: 18446744073709551615L
ERROR: decimal integer literal out of range: 18446744073709551615L
ERROR: decimal integer literal out of range: 18446744073709551615ll
ERROR: decimal integer literal out of range: 18446744073709551615ll
ERROR: decimal integer literal out of range: 18446744073709551615LL
ERROR: decimal integer literal out of range: 18446744073709551615LL
ERROR: Invalid integer literal suffix: 18446744073709551615l_ud1
ERROR: Invalid integer literal suffix: 18446744073709551615L_ud1
ERROR: Invalid integer literal suffix: 18446744073709551615ll_ud1
ERROR: Invalid integer literal suffix: 18446744073709551615LL_ud1
ERROR: decimal integer literal out of range: 18446744073709551614
ERROR: decimal integer literal out of range: 18446744073709551614
ERROR: decimal integer literal out of range: 18446744073709551614l
ERROR: decimal integer literal out of range: 18446744073709551614l
ERROR: decimal integer literal out of range: 18446744073709551614L
ERROR: decimal integer literal out of range: 18446744073709551614L
ERROR: decimal integer literal out of range: 18446744073709551614ll
ERROR: decimal integer literal out of range: 18446744073709551614ll
ERROR: decimal integer literal out of range: 18446744073709551614LL
ERROR: decimal integer literal out of range: 18446744073709551614LL
ERROR: Invalid integer literal suffix: 18446744073709551614l_ud1
ERROR: Invalid integer literal suffix: 18446744073709551614L_ud1
ERROR: Invalid integer literal suffix: 18446744073709551614ll_ud1
ERROR: Invalid integer literal suffix: 18446744073709551614LL_ud1
ERROR: decimal integer literal out of range: 18446744073709551616
ERROR: decimal integer literal out of range: 18446744073709551616
ERROR: decimal integer literal out of range: 18446744073709551616l
ERROR: decimal integer literal out of range: 18446744073709551616l
ERROR: decimal integer literal out of range: 18446744073709551616L
ERROR: decimal integer literal out of range: 18446744073709551616L
ERROR: decimal integer literal out of range: 18446744073709551616ll
ERROR: decimal integer literal out of range: 18446744073709551616ll
ERROR: decimal integer literal out of range: 18446744073709551616LL
ERROR: decimal integer literal out of range: 18446744073709551616LL
ERROR: decimal integer literal out of range: 18446744073709551616u
ERROR: decimal integer literal out of range: 18446744073709551616u
ERROR: decimal integer literal out of range: 18446744073709551616ul
ERROR: decimal integer literal out of range: 18446744073709551616lu
ERROR: decimal integer literal out of range: 18446744073709551616uL
ERROR: decimal integer literal out of range: 18446744073709551616Lu
ERROR: decimal integer literal out of range: 18446744073709551616ull
ERROR: decimal integer literal out of range: 18446744073709551616llu
ERROR: decimal integer literal out of range: 18446744073709551616uLL
ERROR: decimal integer literal out of range: 18446744073709551616LLu
ERROR: decimal integer literal out of range: 18446744073709551616U
ERROR: decimal integer literal out of range: 18446744073709551616U
ERROR: decimal integer literal out of range: 18446744073709551616Ul
ERROR: decimal integer literal out of range: 18446744073709551616lU
ERROR: decimal integer literal out of range: 18446744073709551616UL
ERROR: decimal integer literal out of range: 18446744073709551616LU
ERROR: decimal integer literal out of range: 18446744073709551616Ull
ERROR: decimal integer literal out of range: 18446744073709551616llU
ERROR: decimal integer literal out of range: 18446744073709551616ULL
ERROR: decimal integer literal out of range: 18446744073709551616LLU
ERROR: decimal integer literal out of range: 18446744073709551616l_ud1
ERROR: decimal integer literal out of range: 18446744073709551616L_ud1
ERROR: decimal integer literal out of range: 18446744073709551616ll_ud1
ERROR: decimal integer literal out of range: 18446744073709551616LL_ud1
